### PR TITLE
Apply feed layout design

### LINKED ISSUE #413: [Assigned] Apply feed(information) tab icons
## Overview (Required)
- Only use icon in Zeplin
- You can use reference this.
https://github.com/DroidKaigi/conference-app-2018/pull/396
- There is no need to adjust the layout with this issue

## Links
- https://github.com/DroidKaigi/conference-app-2018/pull/396

If you raise hand, we will send Zeplin Invitation to you

- https://scene.zeplin.io/project/5a327a64f0b20176887c556c/screen/5a527bbce212789e8ea30f1d
- icons: https://app.zeplin.io/project/5a327a64f0b20176887c556c/screen/5a53b2cc33773142c504b949
- screen: https://app.zeplin.io/project/5a327a64f0b20176887c556c/screen/5a523f970721c11708d20d66

### PR DESCRIPTION
**Pull request by sho5nn**
## Issue
- close #413, #414

## Overview (Required)
- Solved #413 and #414 at the same time.

## Links
-

## Screenshot

feed_1

### CODE REVIEW

**Comment by takahirom** [comment]
:cool:

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/feed/item/FeedItem.kt [hunk 1]
```diff
@@ -4,58 +4,68 @@ import android.support.constraint.ConstraintSet
...
 ) {
-    private val touchIgnorer = View.OnTouchListener { _, _ -> true }
+    private val touchIgnore = View.OnTouchListener { _, _ -> true }
+    private var initialized: Boolean = false
+    private var expanded: Boolean = false
 
     override fun bind(viewBinding: ItemFeedBinding, position: Int) {
         viewBinding.post = post
 
+        viewBinding.feedIcon.setImageResource(when (post.type) {
```
Comment (takahirom) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/feed/item/FeedItem.kt]: Can you change the type to enum?
https://github.com/droidkaigi/conference-app-2018/blob/9c045ad14eef92ade7507a660f24156b12d7a110/app/src/main/java/io/github/droidkaigi/confsched2018/data/api/response/mapper/NoticeDataMapperExt.kt#L13-L13

Comment (sho5nn) [reply app/src/main/java/io/github/droidkaigi/confsched2018/presentation/feed/item/FeedItem.kt]: @takahirom How about? 01e18317597b05b5e2c148539eaf52cfecb1cdad

Comment (takahirom) [reply app/src/main/java/io/github/droidkaigi/confsched2018/presentation/feed/item/FeedItem.kt]: GREAT 💯

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/feed/item/FeedItem.kt [hunk 2]
```diff
@@ -4,58 +4,68 @@ import android.support.constraint.ConstraintSet
...
+            Post.Type.Tutorial -> R.drawable.ic_feed_tutorial_pink_20dp
+            Post.Type.Notification -> R.drawable.ic_feed_notification_blue_20dp
+            Post.Type.Alert -> R.drawable.ic_feed_alert_amber_20dp
+            Post.Type.Enquete -> R.drawable.ic_feed_enquete_cyan_20dp
+            else -> R.drawable.ic_feed_notification_blue_20dp
```
Comment (takahirom) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/feed/item/FeedItem.kt]: I think `else` is not needed 👀

Comment (sho5nn) [reply app/src/main/java/io/github/droidkaigi/confsched2018/presentation/feed/item/FeedItem.kt]: oops :P

**Comment by takahirom** [comment]
Looks good! Thank you for smooth animation :+1: :cool:

**Pull request merged**